front_end/src/app/(futureeval)/futureeval/components/orbit/futureeval-orbit.tsx

```
@@ -0,0 +1,237 @@
+      if (animationRef.current) {
+        cancelAnimationFrame(animationRef.current);
+      }
+    };
+  }, [isHovered, expandedItem]);
+
+  // Handle item click actions
+  const handleItemClick = useCallback(
+    (item: OrbitItem) => {
+      switch (item.action.type) {
+        case "scroll":
+          const element = document.getElementById(item.action.target);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)